lib/gap_intelligence/models/merchant_pricing_trend.rb

```diff
@@ -0,0 +1,15 @@
+module GapIntelligence
+  class MerchantPricingTrend < Record
+    def initialize(attrs = {})
+      @raw = attrs
+    end
+
+    def values
```

Comment:
`#values` and `#headers` can be replaced with `attributes :headers, :values, class: Array`